<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeerList</class>
 <widget class="QMainWindow" name="PeerList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meshnet Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListView" name="listView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>269</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionEdit_Config"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_meshnet_client"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Peer"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionEdit_Config">
   <property name="text">
    <string>Edit Config</string>
   </property>
  </action>
  <action name="actionAbout_meshnet_client">
   <property name="text">
    <string>About meshnet-client</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAdd_Peer">
   <property name="text">
    <string>Add Peer</string>
   </property>
   <property name="toolTip">
    <string>Add a new peer</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionDummy">
   <property name="text">
    <string>Dummy</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
